js/ui/calendar/ui.calendar.base_view.js

```diff
@@ -253,36 +277,44 @@ const BaseView = Widget.inherit({
         return this._$table.find(`.${CALENDAR_CONTOURED_DATE_CLASS}`);
     },
 
-    _changeValue: function(cellDate) {
-        if(cellDate) {
-            const value = this.option('value');
-            const newValue = value ? new Date(value) : new Date();
-
-            newValue.setDate(cellDate.getDate());
-            newValue.setMonth(cellDate.getMonth());
-            newValue.setFullYear(cellDate.getFullYear());
-            newValue.setDate(cellDate.getDate());
+    _renderValue: function() {
+        if(!this.option('allowValueSelection')) {
+            return;
+        }
 
-            this.option('value', newValue);
-        } else {
-            this.option('value', null);
+        let value = this.option('value');
+        if(!Array.isArray(value)) {
+            value = [value];
         }
+
+        this._selectedCells?.forEach((cell) => (cell.removeClass(CALENDAR_SELECTED_DATE_CLASS)));
```

Comment:
[Airbnb guide - arrow functions](https://github.com/airbnb/javascript#arrow-functions)
Let's fix it everywhere
```suggestion
        this._selectedCells?.forEach((cell) => { cell.removeClass(CALENDAR_SELECTED_DATE_CLASS); });
```

Reply:
done